src/classes/RLLP_OppRollup.cls

```diff
@@ -363,7 +363,13 @@ public without sharing class RLLP_OppRollup {
             }
         }         
 
-        update accountsToUpdate.values();     
```

Comment:
@randi274  The 3 sections in this class and the one in RLLP_OppPartialSoftCreditRollup that do the update and error processing are all nearly identical. Is it worth creating a single method in UTIL_DMLService for `updateRecordsAndLog(List<SObject> records, ERR_Handler_API.Context context)` that each of these can just call with the list of records?

Reply:
Good point, I can use that in the soft credit rollup as well!